packages/eslint-config-react/index.js

```diff
@@ -1,269 +1,352 @@
 /**
  * strvcom/eslint-config-react
  *
- * @author      Robert Rossmann <rr.rossmann@me.com>
+ * @author      Pavel Prichodko <pavel.prichodko@strv.com>
  * @copyright   2019 STRV
  * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
  */
 
 'use strict'
 
 module.exports = {
+  root: true,
+  extends: ['@strv/eslint-config-base'],
 
-  extends: require.resolve('@strv/eslint-config-base'),
+  parser: 'babel-eslint',
```

Comment:
Unfortunately, it looks like it is necessary. Without it we are getting errors from named imports because parser doesn't understand them by default. 

It looks like it works just with `.mjs` extension, which is something we don't want to use.

![image](https://user-images.githubusercontent.com/2115953/55627022-70816e00-57ad-11e9-94a4-be29d2dcfbde.png)
![image](https://user-images.githubusercontent.com/2115953/55627086-9c045880-57ad-11e9-879a-84e41d08db6d.png)

Reply:
That is a valid point, but I feel that the parser's choice should be left to the end user and not bundled with a ruleset, because the compiler (webpack, babel etc.) is also managed by the end user. When a new JavaScript feature is implemented in one of these tools it should be easy for the end user to also update the parser (or choose a completely different one) to get that feature supported in ESLint as well.

Let's remove the parser from this ruleset please. 🙏 

> PS. I use this parser on almost all of my projects, but I'd prefer to have them installed as a direct dependency rather than through a ruleset.

Reply:
We agreed to keep the parser here. ✅